common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -463,12 +464,13 @@ class @Problem
       saveMessage = response.msg
       if response.success
         @el.trigger('contentChanged', [@id, response.html])
-        @el.find('.notification-save .notification-message').html(saveMessage)
+        edx.HtmlUtils.setHtml(@el.find('.notification-save .notification-message'), edx.HtmlUtils.HTML(saveMessage))
         @clear_all_notifications()
         @saveNotification.show()
         @focus_on_save_notification()
       else
         @gentle_alert saveMessage
+      @updateProgress response
```

Comment:
I think we removed this originally in our work because we couldn't figure out what "progress" needed to be updated. Do you understand it now (and can you share with me)?